File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    // Declare num
    int num = 0;
    
    // Prompt for height
    do 
    {
        num = get_int("Height: \n");
    }
    while (num < 1 || num > 8);
    
    // Make pyramid
    for (int i = 0; i < num; i++)
    {
        // Spaces before left pyramid
        for (int k = 0; k < num - i - 1; k++)
        {
            printf(" ");
        }
        
        // Left pyramid
        for (int j = 0; j <= i; j++) 
        {
            printf("#");
        }
        
        // spaces in between
        printf("  ");
        
        // Right pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

num = 0
while num < 1 or num > 8:
    num = get_int("Height: ")

for i in range(num):
    # spaces before
    for j in range(0, num - i - 1, 1):
        print(" ", end = "")
        
    #pyramid
    for k in range(0, i + 1, 1):
        print("#", end = "")
        
    print()